pkg/edgeview/src/tcp.go

```diff
@@ -72,7 +72,9 @@ type tcpConnRWMap struct {
 }
 
 const (
-	tcpMaxMappingNUM int = 5
+	tcpMaxMappingNUM  int           = 5
+	tcpIdleTimeoutSec float64       = 1800.0
+	tcpCheckTimeout   time.Duration = 300
```

Comment:
This one still reads like nanoseconds. Makes more sense to move the multiplication with time.Seconds here and not have it on line 337

Reply:
Sure.